pkg/setting/setting.go

```diff
@@ -1181,6 +1181,8 @@ func (cfg *Cfg) parseINIFile(iniFile *ini.File) error {
 	cfg.Env = valueAsString(iniFile.Section(""), "app_mode", "development")
 	cfg.StackID = valueAsString(iniFile.Section("environment"), "stack_id", "")
 	cfg.Slug = valueAsString(iniFile.Section("environment"), "stack_slug", "")
+	// This simply identifies an instance running in Cloud, and is NOT for controlling enablement of Cloud features on-prem.
+	cfg.IsCloud = iniFile.Section("environment").Key("is_cloud").MustBool(false)
```

Comment:
StackID only exists for our single tenant pods running in cloud (it is invalid anywhere else)

would this be different than a property that is `stack_id` != ""